packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -381,6 +381,23 @@ describeFullCompat("Flushing ops", (getTestObjectProvider) => {
         });
     });
 
+    describe("Batch validation when using getPendingLocalState()", () => {
+        beforeEach(async () => {
+            await setupContainers({ flushMode: FlushMode.TurnBased, enableOfflineLoad: true });
+        });
+        it("should not allow incomplete batch during orderSequentially", async () => {
```

Comment:
#nit the test description is very ambiguous. Should be something like 'cannot capture the pending local state during ordersequentially'